features/environment.py

```diff
@@ -848,6 +848,11 @@ def _install_uat_in_container(
             )
         )
 
+    if config.enable_proposed:
+        cmds.append(
+            ["sudo", "rm", "/etc/apt/sources.list.d/uaclient-proposed.list"]
```

Comment:
Should we run also `apt update` after this, just to make sure?

Reply:
Ah good catch - yeah we probably should.

Reply:
Just added this